wheel/chia_rs.pyi

```diff
@@ -55,19 +55,19 @@ NO_UNKNOWN_OPS: int = ...
 
 def run_chia_program(
     program: bytes, args: bytes, max_cost: int, flags: int
-) -> Pair[int, LazyNode]: ...
+) -> Tuple[int, LazyNode]: ...
 
 class LazyNode:
-    def pair() -> Optional[Tuple[LazyNode, LazyNode]]: ...
-    def atom() -> bytes: ...
+    pair: Optional[Tuple[LazyNode, LazyNode]]
+    atom: bytes
```

Comment:
actually, it looks like this should be `Optional[bytes]`. Did some tool detect this and say it should be just `bytes`?

Reply:
ok I'll fix that in a bit, the generate type stubs script hard codes this (another reason automatic generation would be nice for everything later)